tip-wlan/Chart.yaml

```diff
@@ -26,54 +26,55 @@ sources:
-  - name: opensync-mqtt-broker
-    version: 0.1.0
-    condition: opensync-mqtt-broker.enabled
-  - name: wlan-cloud-graphql-gw
-    version: 0.1.0
-    condition: wlan-cloud-graphql-gw.enabled
-  - name: wlan-cloud-static-portal
-    version: 0.1.0
-    condition: wlan-cloud-static-portal.enabled
+- name: opensync-gw-static
+  version: 0.1.0
+  condition: opensync-gw-static.enabled
+- name: opensync-gw-cloud
+  version: 0.1.0
+  condition: opensync-gw-cloud.enabled
+- name: opensync-mqtt-broker
+  version: 0.1.0
+  condition: opensync-mqtt-broker.enabled
+- name: wlan-cloud-graphql-gw
+  version: 0.1.0
+  condition: wlan-cloud-graphql-gw.enabled
+- name: wlan-cloud-static-portal
+  version: 0.1.0
+  condition: wlan-cloud-static-portal.enabled
 ## IntegratedCloudComponent should be disabled for distributed deployment
-  - name: wlan-integrated-cloud-component-service
-    version: 0.1.0
-    condition: wlan-integrated-cloud-component-service.enabled
-  - name: wlan-portal-service
-    version: 0.1.0
-    condition: wlan-portal-service.enabled
-  - name: wlan-prov-service
-    version: 0.1.0
-    condition: wlan-prov-service.enabled
-  - name: wlan-ssc-service
-    version: 0.1.0
-    condition: wlan-ssc-service.enabled
-  - name: wlan-spc-service
-    version: 0.1.0
-    condition: wlan-spc-service.enabled
-  - name: wlan-port-forwarding-gateway-service
-    version: 0.1.0
-    condition: wlan-port-forwarding-gateway-service.enabled
-  - name: nginx-ingress-controller
-    version: 0.1.0
-    condition: nginx-ingress-controller.enabled
-  - name: common
-    version: 0.1.0
-  - name: zookeeper
-    version: 0.1.0
-    condition: zookeeper.enabled
-  - name: kafka
-    version: 0.1.0
-    condition: kafka.enabled
-  - name: cassandra
-    version: 0.1.0
-    condition: cassandra.enabled
-  - name: postgresql
-    version: 8.9.6
-    condition: postgresql.enabled
\ No newline at end of file
+- name: wlan-integrated-cloud-component-service
+  version: 0.1.0
+  condition: wlan-integrated-cloud-component-service.enabled
+- name: wlan-portal-service
+  version: 0.1.0
+  condition: wlan-portal-service.enabled
+- name: wlan-prov-service
+  version: 0.1.0
+  condition: wlan-prov-service.enabled
+- name: wlan-ssc-service
+  version: 0.1.0
+  condition: wlan-ssc-service.enabled
+- name: wlan-spc-service
+  version: 0.1.0
+  condition: wlan-spc-service.enabled
+- name: wlan-port-forwarding-gateway-service
+  version: 0.1.0
+  condition: wlan-port-forwarding-gateway-service.enabled
+- name: nginx-ingress-controller
+  version: 0.1.0
+  condition: nginx-ingress-controller.enabled
+- name: common
+  version: 0.1.0
+- name: zookeeper
+  version: 0.1.0
+  condition: zookeeper.enabled
+- name: kafka
+  version: 0.1.0
+  condition: kafka.enabled
+- name: postgresql
+  version: 10.1.0
+  repository: https://charts.bitnami.com/bitnami
+  condition: postgresql.enabled
+- name: cassandra
+  version: 5.5.3
+  condition: cassandra.enabled
```

Comment:
Are we missing repository here?

Reply:
this is a postgres PR, cassandra PR has this.